client/galaxy/scripts/mvc/grid/grid-template.js

```diff
@@ -92,8 +92,8 @@ export default {
         for (let column of options.columns) {
             if (column.visible) {
                 tmpl += `<th id="${column.key}-header">`;
-                if (column.href) {
-                    tmpl += `<a href="${column.href}" class="sort-link" sort_key="${column.key}">${column.label}</a>`;
+                if (column.sortable) {
+                    tmpl += `<a href="javascript:void(0)" class="sort-link" sort_key="${column.key}">${column.label}</a>`;
```

Comment:
If we cannot give it a reasonable anchor/push url  I suggest we use `href="#"` and `event.preventDefault()` instead of introducing a new way how to do this into our codebase.

Reply:
There are 6 other locations in the same file which handle it like that.

Reply:
You are right, my search-fu failed. Sorry for the noise.

Reply:
No worries, Ill remove my last commit then.